CHANGELOG.md

```diff
@@ -3,6 +3,73 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.1.0
+
+_Nov 2, 2021_
+
+A big thanks to the 11 contributors who made this release possible. Here are some highlights ✨:
+
+### `@mui/x-data-grid@v5.1.0` / `@mui/x-data-grid-pro@v5.1.0`
+
+- 🚀 Introduce the [tree data](https://mui.com/components/data-grid/group-pivot/#tree-data) feature (#2725) @flaviendelangle
+
+  <img src="https://user-images.githubusercontent.com/42154031/144259216-fc4f90ac-4d8b-4253-bc95-009204349a4c.gif" width="854" height="453" />
+
+- 💅 Add support for `sx` prop in the DataGrid and DataGridPro (#3281) @m4theushw
+- 🔦 Improve focus management in the filter panel (#3004) @alexfauquette
+- 🌎 Translation updates for many locales
+
+  If you are using DataGrid or DataGridPro in another language, check [this issue](https://github.com/mui-org/material-ui-x/issues/3211) to discover which translations are missing.
+
+- 📚 Documentation improvements
+- 🐞 Bugfixes
+
+#### Changes
+
+- [DataGridPro] Add tree data (#2725) @flaviendelangle
+- [DataGridPro] Improve license file (#3278) @oliviertassinari
+- [DataGridPro] Add types for event publishers and listeners (#3022) @flaviendelangle
```

Comment:
You have the highlight for this one in the description of the PR.

Reply:
Updated.